distributed/scheduler.py

```diff
@@ -2819,15 +2820,15 @@ def is_rootish(self, ts: TaskState) -> bool:
         Root-ish tasks are part of a group that's much larger than the cluster,
         and have few or no dependencies.
         """
-        if ts.resource_restrictions or ts.worker_restrictions or ts.host_restrictions:
+        if (
+            ts.resource_restrictions
+            or ts.worker_restrictions
+            or ts.host_restrictions
+            or ts.actor
+        ):
             return False
         tg = ts.group
-        # TODO short-circuit to True if `not ts.dependencies`?
```

Comment:
If this all works out well, I would actually like to raise this method to a `TaskGroup` property instead. We can implement a similar short-circuit there although I'm not sure if this is actually worth it.